src/core/build_env.go

```diff
@@ -232,6 +232,13 @@ func runtimeDataPaths(graph *BuildGraph, data []BuildInput) []string {
 func RunEnvironment(state *BuildState, target *BuildTarget, inTmpDir bool) BuildEnv {
 	env := TargetEnvironment(state, target)
 
+	outEnv := target.GetTmpOutputAll(target.Outputs())
```

Comment:
I'm not sure `GetTmpOutputAll` is the right thing here; we're going to be using the real outputs, not temp names?

Reply:
Ya, I guess `GetTmpOutputAll` is meant for the build env, not the run env.

Reply:
Yep exactly, it handles some cases where your output has the same name as a top-level source directory.

In hindsight it would have been better to have outputs be in their package directory rather than at the top level, but it's way too hard to change now :(